### PR TITLE
Unit tests for Homepage from signed-out perspective

### PR DESCRIPTION
**Pull request by Khaalid-oh**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

Screenshot 2024-01-01 at 15 54 48

### CODE REVIEW

**Comment by elraphty** [comment]
@Khaalid-oh Please also add a limit to the pagination mocked API requests ```/api/bounties?page=2&limit=20```

File: frontend/app/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "@mui/system": "^5.14.10",
     "add": "^2.0.6",
     "bootstrap": "^4.5.0",
+    "chai": "^5.0.0",
```
Comment (kevkevinpal) [inline frontend/app/package.json:39]: why did we add this dependency `jest` already has `expect`

**Comment by kevkevinpal** [comment]
I dont see how this is a component test at all??? I don't see any components being rendered in the tests

you are just mocking an API request then asserting that you get the same mock which is not testing anything

**Comment by Khaalid-oh** [comment]
The need for Sinon.js for stub of the real function made me assume the Unit test was more aligned with an integration test for an API endpoint rather than a component test.

**Comment by Khaalid-oh** [comment]
> @Khaalid-oh Please also add a limit to the pagination mocked API requests ```/api/bounties?page=2&limit=20```

Unassigned

**Pull request closed without merging**